dll/cpl/desk/effappdlg.c

```diff
@@ -70,7 +68,7 @@ AddToCombobox(INT Combo, HWND hwndDlg, INT From, INT To)
 
     for (iElement = From; iElement <= To; iElement++)
     {
-        LoadString(hApplet, iElement, (LPTSTR)tstrText, ARRAYSIZE(tstrText));
```

Comment:
Why? What does that fix? How is that better?

Reply:
It's a character shorter, and until now nobody found anything that would be worse after the change. Up to now ARRAYSIZE vs _countof is used in mixed manner within this module. Afterwards _countof() is used everywhere. If you have any concerns regarding the change, write out your concerns please in letters.